### ISSUE TITLE
Deep Learning for Recommender Systems

### ISSUE THREAD

**Issue by PXIONG001** [PXIONG001/Research#32]
### Title
Deep Learning for Recommender Systems
### URL
https://dl.acm.org/doi/10.1145/3109859.3109933
### Summary 
Deep Learning is one of the next big things in Recommendation Systems technology. The past few years have seen the tremendous success of deep neural networks in a number of complex machine learning tasks such as computer vision, natural language processing and speech recognition. After its relatively slow uptake by the recommender systems community, deep learning for recommender systems became widely popular in 2016. We believe that a tutorial on the topic of deep learning will do its share to further popularize the topic. Notable recent application areas are music recommendation, news recommendation, and session-based recommendation. The aim of the tutorial is to encourage the application of Deep Learning techniques in Recommender Systems, to further promote research in deep learning methods for Recommender Systems.
### Key Points 
1. Deep Learning (Deep Neural Networks)
2. Matrix Factorization
3. Tensor Factorization
4. Gradient Descent for Optimization
